lib/ansible/modules/cloud/vmware/vmware_dns_config.py

```diff
@@ -1,123 +0,0 @@
-#!/usr/bin/python
```

Comment:
We can not directly remove module file. First we need to add deprecation note like this - https://github.com/ansible/ansible/blob/dc2eb0549168f55a66084a660043d477c6edf30f/lib/ansible/modules/cloud/vmware/_vsphere_guest.py#L18

Reply:
This was the procedure @dagwieers told me to use.

Reply:
We are not removing it, we are renaming it to another module. Which is something we could not do with **vsphere_guest** because **vmware_guest** is not a drop-in replacement.

Reply:
How about we deprecate `vmware_dns_config` and make `vmware_host_dns` as drop-in replacement.

Reply:
@Akasurde That's exactly what we are doing.

Reply:
@Akasurde I assume the procedure is ok?